dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -28,26 +28,35 @@ PROVIDES="
 	cmd:valac_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapicheck_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapigen_${portVersion%.*}$secondaryArchSuffix
-	lib:libvala_${portVersion%.*}$secondaryArchSuffix
-	"
-PROVIDES_common="
-	vala${secondaryArchSuffix}_common = $portVersion
+	lib:libvala_${portVersion%.*}$secondaryArchSuffix = 0.0.0 compat >= 0
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	glib2${secondaryArchSuffix} >= 2.32.0
 	vala${secondaryArchSuffix}_common == $portVersion
+	lib:libdbus_1$secondaryArchSuffix
 	lib:libffi$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libintl$secondaryArchSuffix
 	"
+PROVIDES_common="
+	vala${secondaryArchSuffix}_common = $portVersion
+	"
 REQUIRES_common="
 	haiku$secondaryArchSuffix
 	"
+PROVIDES_devel="
+	vala${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	#devel:libvala_${portVersion%.*}$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+REQUIRES_devel="
+	vala$secondaryArchSuffix == $portVersion base
+	"
```

Comment:
Because of this:
```bash
grep Requires /system/develop/lib/pkgconfig/libvala-0.36.pc 
Requires: glib-2.0 gobject-2.0
```
I think it might be appropriate to mention glib and gobject in `REQUIRES_devel`:
```diff
 REQUIRES_devel="
 	vala$secondaryArchSuffix == $portVersion base
+	devel:libglib_2.0
+	devel:libgobject_2.0
 	"
```
(But I'l not sure about this, as there could be downsides.)
Let's see what other contributors think about this.

Reply:
Been looking into updating cairo, this can also use glib2 (which in it turn produces libcairo-gobject, which could be used to fix gobject-introspection tests) :)

Reply:
PS, haven't seen any lib so far using devel:* from other libraries aside from it's own?